ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -45,7 +45,9 @@ tests:
                   --github-app-key /secrets/rebasebot/cloud-team-openshift-bot-key \
                   --github-cloner-key /secrets/rebasebot/cloud-team-bot-key \
                   --git-username cloud-team-rebase-bot \
-                  --git-email cloud-team-rebase-bot@redhat.com
+                  --git-email cloud-team-rebase-bot@redhat.com \
+                  --bot-emails cloud-team-rebase-bot@redhat.com openshift-bot@redhat.com openshift-ci-robot@redhat.com \
```

Comment:
does this value need some sort of separator?

Reply:
Nope. This should be the way to use that flag.

Reply:
looks like maybe not, https://docs.python.org/3/library/argparse.html#nargs